src/server.js

```diff
@@ -99,10 +99,10 @@ app.use('/nso-legacy-pack', routes.aprilfools);
 logger.info('Creating 404 status handler');
 // This works because it is the last router created
 // Meaning the request could not find a valid router
-app.use((request, response, next) => {
+app.use((request, response) => {
 	const fullUrl = util.fullUrl(request);
+	response.render('404');
 	logger.warn(`HTTP 404 at ${fullUrl}`);
```

Comment:
This isn't your code, but this logger call can be removed now I think. It's not super useful and this PR is a good time to remove it